packages/common/src/utils/misc.js

```diff
@@ -1,6 +1,21 @@
+import { Buffer } from "buffer";
```

Comment:
@werme should I add this import into the zlibcompress as well?

Reply:
Yeah preferrably. If you had a working version with `atob` I’d recommend that tho to hold back on dependencies.

Reply:
You can polyfill `atob` on react native with `base-64` from npm (https://stackoverflow.com/questions/42829838/react-native-atob-btoa-not-working-without-remote-js-debugging#answer-51525605)